source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -170,7 +170,8 @@ Decommissioning Ignores Expired Objects and Trailing ``DeleteMarker``
 Starting with :minio-release:`RELEASE.2023-05-27T05-56-19Z`, decommissioning ignores objects where the only remaining version is a ``DeleteMarker``.
 This avoids creating empty metadata on the remaining server pool(s) for objects that are effectively fully deleted.
 
-Starting with :minio-release:`minio-lifecycle-management-scanner`, decommissioning also ignores object versions which have expired based on the configured :ref:`lifecycle rules <minio-lifecycle-management-expiration>` for the parent bucket.
+Starting with :minio-release:`RELEASE.2023-06-23T20-26-00Z`, decommissioning also ignores object versions which have expired based on the configured :ref:`lifecycle rules <minio-lifecycle-management-expiration>` for the parent bucket.
+Starting with :minio-release:`RELEASE.2023-06-29T05-12-28Z`, decommissioning logs ignored delete markers and expired objects.
```

Comment:
When a delete marker or expired object is skipped during decommissioning, is its name logged as well? From the diff, it looks like a generic message that something was skipped. I'm not sure if the name is added to the message elsewhere.
@harshavardhana

Reply:
We do not as in in a log we send a trace if someone is listening via `mc admin trace --call decom`

Reply:
Ok so this isn't logging at all. But it's viewable by tracing that call type (along with whatever was there already). 👍🏻